nomad/local/local_grapl_registry.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -euo pipefail
+
+GRAPL_LOCAL_REGISTRY_PORT=5000
+GRAPL_LOCAL_REGISTRY_NAME="grapl_local_registry"
+
+if [[ ! -z "$(docker ps --quiet --filter name="${GRAPL_LOCAL_REGISTRY_NAME}")" ]]; then
```

Comment:
It would be simpler to use `-n` ("string is not null") than `! -z` (NOT "string is null")

Reply:
This should be corrected now, actually. I made this change as a result of the formatter and pushed the change up before standup.

Reply:
cool 👍